contracts/Comet.sol

```diff
@@ -592,6 +592,9 @@ contract Comet is CometMainInterface {
      * @dev Note: The assumption `newPrincipal >= oldPrincipal` MUST be true
```

Comment:
I think we can remove this now, or move the comment below to a dev note?

Reply:
Done.